drivers/SmartThings/zigbee-switch/fingerprints.yml

```diff
@@ -1966,6 +1966,23 @@ zigbeeManufacturer:
     manufacturer: GE
     model: Daylight
     deviceProfileName: ge-link-bulb
+  - id: "Insta GmbH/Switching Actuator"
+    deviceLabel: NEXENTRO Switching Actuator
+    manufacturer: Insta GmbH
+    model: NEXENTRO Switching Actuator
+    deviceProfileName: basic-switch
+  - id: "Insta GmbH/Dimming Actuator"
+    deviceLabel: NEXENTRO Dimming Actuator
+    manufacturer: Insta GmbH
+    model: NEXENTRO Dimming Actuator
+    deviceProfileName: on-off-level
```

Comment:
you have doubled the dimming actuator fingerprint

Reply:
I am not sure what you mean. The switching Actuator and the dimming Actuator are two separate devices.

Reply:
Oh. Sorry. I misread the code.